cloud/azure/custom/api.pm

```diff
@@ -334,7 +334,7 @@ sub azure_get_resource_health_set_url {
 
     my $url = $self->{management_endpoint} . "/subscriptions/" . $self->{subscription} . "/resourceGroups/" .
         $options{resource_group} . "/providers/" . $options{resource_namespace} . "/" . $options{resource_type} .
-        "/" . $options{resource} . "/providers/Microsoft.ResourceHealth/availabilityStatuses/current?api-version=" . $self->{api_version};
+        "/" . $options{resource} . "/providers/Microsoft.ResourceHealth/availabilityStatuses/current?api-version=" . $options{api_version};
```

Comment:
@thibaults-centreon , could you explain why this is needed? I'm probably missing something.

Reply:
the ResourceHealth endpoint needs a specific API version which is hardcoded here: https://github.com/centreon/centreon-plugins/blob/master/cloud/azure/management/monitor/mode/health.pm#L116
Then the value has to be fetched from the options.

Reply:
Heyhey of course. Thanks !